### PR TITLE
Adding a channel for scans.

### PR DESCRIPTION
**Pull request by sduskis**
There was a problem doing mutations while a scan was ongoing.  Adding a channel specifically for scans.

### CODE REVIEW

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableGrpcClient.java [hunk 1]
```diff
@@ -140,15 +139,18 @@ public void onCompleted() {
...
   private final RetryOptions retryOptions;
 
   public BigtableGrpcClient(
-      Channel channel,
+      Channel generalChannel,
+      Channel scanChannel,
       ExecutorService executorService,
       RetryOptions retryOptions) {
-    this.channel = channel;
+    this.generalChannel = generalChannel;
```
Comment (kevinsi4508) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableGrpcClient.java]: how about following?
readChannel
writeChannel?

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java [hunk 2]
```diff
@@ -271,8 +271,10 @@ public void run() {
 
   public synchronized BigtableClient getDataClient() throws IOException {
     if (this.client == null) {
-      Channel channel = createChannel(options.getDataHost(), options.getChannelCount());
-      this.client = new BigtableGrpcClient(channel, batchPool, options.getRetryOptions());
+      Channel generalChannel = createChannel(options.getDataHost(), options.getChannelCount());
+      Channel scanChannel = createChannel(options.getDataHost(), 1);
```
Comment (kevinsi4508) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: So for read, 1 channel is enough?

Comment (sduskis) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: Yes.  Scans run a long time over a single channel.  We encourage a lot of mutations asynchronously, so mutations benefit from more channels.

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableGrpcClient.java (hunk 1 above)
Comment (sduskis) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableGrpcClient.java]: that's reasonable.

**Comment by sduskis** [comment]
Renamed the channels and added a comment in BigtableGrpcClient.

**Comment by kevinsi4508** [comment]
LGTM

**Pull request merged**